client.go

```diff
@@ -55,6 +56,10 @@ type Client struct {
 	compression proto.Compression
 
 	settings []Setting
+
+	// SSH authentication
+	sshKey *ssh.SSHKey
+	useSSH bool
```

Comment:
Looks like `useSSH` == `sshKey != nil` and is excessive, we can just leave `sshKey` here.